manifests/update.pp

```diff
@@ -56,8 +56,9 @@
   } else {
     $_refresh = true
   }
+  $command = [$apt::provider, 'update']
   exec { 'apt_update':
-    command     => "${apt::provider} update",
+    command     => $command,
```

Comment:
Would be better to put the array as the parameter, no need for the `$command` variable.